docs/user-guide/helm.md

```diff
@@ -50,6 +50,21 @@ source:
     - values-production.yaml
 ```
 
+ArgoCD now supports Glob expressions for valueFiles (especially useful when using umbrella charts with a lot of dependencies) :
+
+```bash
+argocd app set helm-guestbook --values subdir/**/values-*.yaml
+```
+
+In the declarative syntax:
+
+```yaml
+source:
+  helm:
+    valueFiles:
+    - subdir/**/values-*.yaml
```

Comment:
I think we should document ordering. If I'm reading the code correctly, we sort per-glob and then preserve `valueFiles` list order. Is that right?

Reply:
I added theses precisions in my latest commit